tutorials/llm/llama-3/pruning-distillation/05_display_results.ipynb

```diff
@@ -0,0 +1,168 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6c91263b-b312-4ab2-b13f-0ee4b6e8bd0f",
+   "metadata": {},
+   "source": [
+    "### Step 5: Display the validation loss\n",
+    "\n",
+    "Now that the results are in, let's visualize the validation loss of the two distilled models using the `tensorboard` library. \n",
+    "> `NOTE:` This notebook demonstrates the use of the teacher finetuning, pruning and the distillation script. These scripts should ideally be run on a multi-node cluster with a larger `GLOBAL_BATCH_SIZE` and `STEPS` to see improvement in the validation loss."
```

Comment:
fix punctuation

`NOTE:` This notebook demonstrates the use of the teacher fine-tuning, pruning, and the distillation script. These scripts should ideally be run on a multi-node cluster with a larger `GLOBAL_BATCH_SIZE` and `STEPS` to see improvement in the validation loss."